dask/dataframe/core.py

```diff
@@ -1169,11 +1270,13 @@ def repartition(
         ):
             raise ValueError(
                 "Please provide exactly one of ``npartitions=``, ``freq=``, "
-                "``divisisions=``, ``partitions_size=`` keyword arguments"
+                "``divisions=``, ``partitions_size=``, ``partition_sizes=`` keyword arguments"
```

Comment:
We have `partition_sizes` (plural) b/c it can contain the series of exact sizes, whereas `partition_size` has the default size, but the final partition might be smaller, and mapping across partitions could cause other kinds of alternate sizes.